content/learn/05.communication/07.modbus/modbus.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Modbus"
+description: "Modbus is an open serial communication protocol used for transmitting information over serial lines between electronic devices."
+author: "Hannes Siebeneicher"
+---
+
```

Comment:
I would add a general introduction to what this article will contain. Basically, it is guide to understanding how modbus can be used with Arduino hardware. (and after this, add "compatible hardware" as I wrote in the other comment)